packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapHeatmapController.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+@import GoogleMapsUtils;
+
+// Defines heatmap UI options writable from Flutter.
+@protocol FLTGoogleMapHeatmapOptionsSink
+- (void)setWeightedData:(NSArray<GMUWeightedLatLng *> *)weightedData;
```

Comment:
Please add an extra line better class/protocol definition and method signatures, and between method signatures and @end for easier reading. ditto below.

Reply:
I understand you copied the existing code. I think this is ok, i'll resolve this comment.